Cabal/doc/developing-packages.rst

```diff
@@ -1147,6 +1147,83 @@ For example, given the following dependencies specified in
       foo >= 0.5.2 && < 0.6
       bar >= 1.1 && < 1.2
 
+Listing outdated dependency version bounds
+""""""""""""""""""""""""""""""""""""""""""
+
+Manually updating dependency version bounds in a ``.cabal`` file or a
+freeze file can be tedious, especially when there's a lot of
+dependencies. The ``cabal outdated`` command is designed to help with
+that. It will print a list of packages for which there is a new
+version on Hackage that is outside the version bound specified in the
+``build-depends`` field. The ``outdated`` command can also be
+configured to act on the freeze file (both old- and new-style) and
+ignore major (or all) version bumps on Hackage for a subset of
+dependencies.
+
+The following flags are supported by the ``outdated`` command:
+
+``--freeze-file``
+    Read dependency version bounds from the freeze file (``cabal.config``)
+    instead of the package description file (``$PACKAGENAME.cabal``).
+``--new-freeze-file``
+    Read dependency version bounds from the new-style freeze file
+    (``cabal.project.freeze``) instead of the package description file.
+``--simple-output``
+    Print only the names of outdated dependencies, one per line.
+``--exit-code``
+    Exit with a non-zero exit code when there are outdated dependencies.
+``-q, --quiet``
+    Don't print any output. Implies ``-v0`` and ``--exit-code``.
+``--ignore`` *PACKAGENAMES*
```

Comment:
Actually no, that form isn't supported, because ignoring everything doesn't make sense (at least until a whitelist feature is added).